lhalley-schaumburg.md

```diff
@@ -0,0 +1,6 @@
+# Schaumburg, IL
+
+### Restaurants
+- Texas do Brasil
+### Attractions
```

Comment:
I like :hamburger: from Portillos, and some :kr:  food.

Reply:
Added some restaurants, let me know if it's ok to ship